db/migrate/20181229111736_add_admin_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
Please add `null: false` to boolean columns, if you don't need to have `NULL` value in database.